### PR TITLE
Add minfontsize and maxfontsize attributes to provide bounds on fontsize

### PR DESCRIPTION
**Pull request by fjvallarino**
## Description
Adds two attributes to control the accepted min and max values for `fontsize`.

This PR replaces the previous one based off `feature/fontsize-limits`. The reason for closing the other PR was to only keep one branch updated.

## Motivation and Context
It was requested by Creative team.

## How Has This Been Tested?
Unit tests have been added.

It can also be tested here: https://apps-stage-7.risevision.com/templates/edit/3ac7b6b2-8280-40a2-8e10-60489e702728/?cid=87977ab8-38b6-47fb-ad5e-256b8cc4b46d

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No.

@santiagonoguez @stulees please review

### CODE REVIEW

**Comment by santiagonoguez** [comment]
though coveralls seem to be hit :O

**Comment by fjvallarino** [comment]
@santiagonoguez yeah, I could not find a way for that line to be covered. It's a switch that's being called with two different values in different tests, but for some reason it does not mark it as covered. If you have any suggestion, let me know and I'll apply it!

File: src/rise-text.js [hunk 1]
```diff
@@ -3,6 +3,9 @@ import "@polymer/polymer/lib/elements/dom-if.js";
 import { RiseElement } from "rise-common-component/src/rise-element.js";
 import { version } from "./rise-text-version.js";
 
+const MIN_TEXT_SIZE = 1;
```
Comment (stulees) [inline src/rise-text.js]: @fjvallarino We've been declaring constants with `static GET` within the class. Declaring them as you are here is going to make them global. See https://github.com/Rise-Vision/rise-time-date/blob/master/src/rise-time-date.js#L67-L89 for example. Can you set the `value` of the properties directly and then declare the constants as per usual to use in the rest of the class?

Comment (fjvallarino) [reply src/rise-text.js]: @stulees but those constants (`MIN_TEXT_SIZE` and `MAX_TEXT_SIZE`) are not exported. I think you need to explicitly set `global.MIN_TEXT_SIZE = 1` for them to be global.

Comment (stulees) [reply src/rise-text.js]: @fjvallarino Okay, was preferring to stay consistent between our components and declaration of constants, but I'm not feeling that strongly. Thanks

**Pull request merged**